test/shared/TestServiceContext.cs

```diff
@@ -32,7 +33,11 @@ public TestServiceContext(ILoggerFactory loggerFactory, IKestrelTrace kestrelTra
             HttpParserFactory = frameAdapter => new HttpParser<FrameAdapter>(frameAdapter.Frame.ServiceContext.Log.IsEnabled(LogLevel.Information));
             ServerOptions = new KestrelServerOptions
             {
-                AddServerHeader = false
+                AddServerHeader = false,
+                Limits =
+                {
+                    MinResponseDataRate = new MinDataRate(bytesPerSecond: 1, gracePeriod: TimeSpan.MaxValue)
```

Comment:
This is super scary. If we have to do this, let's try to make this change to specific tests.